inst/rstudio/templates/project/teal_application.dcf

```diff
@@ -0,0 +1,9 @@
+Binding: project_skeleton
+Title: Teal Application
+OpenFiles: app.R
+Icon: teal.png
+
+Parameter: use_cdisc
+Widget: CheckboxInput
```

Comment:
(in the future) Can the Widget be for example?
```
Widget: availableModulesInput
```

Reply:
you can only have the following:

![image](https://user-images.githubusercontent.com/15201933/180205738-4091c547-c140-4b17-82e9-b7ac5cca4ef6.png)

We may want an addin to do the add modules part rather than at this project creation time. And for creating your own custom modules you could maybe? use [snippets](https://rstudio.github.io/rstudio-extensions/rstudio_snippets.html)

Reply:
Is snippet a Rstudio-exclusive feature?

Reply:
I don't think snippet is RStudio exclusive feature (i.e. you can add code snippet on vscode too). However, the syntax will different.